control/control.CASP.xml

```diff
@@ -314,44 +295,41 @@ Please visit us at http://www.suse.com/.
         <service_sfcb><label>CIM Server</label></service_sfcb>
         <roles_caption>
           <!-- TRANSLATORS: dialog caption -->
-          <label>System Role</label>
+          <label>CASP Roles</label>
         </roles_caption>
         <roles_text>
           <!-- TRANSLATORS: label in a dialog -->
-          <label>System Roles are predefined use cases which tailor the system
+          <label>Roles are predefined use cases which tailor the system
 for the selected scenario.</label>
         </roles_text>
         <roles_help>
           <!-- TRANSLATORS: dialog help -->
-          <label>&lt;p&gt;The system roles adjustments are in the range from package selection up 
-to disk partitioning. By choosing a system role, the system is 
-configured accordingly to match the use case of the role. The settings 
-defined by a role can be overridden in the next steps if necessary.&lt;/p&gt;</label>
+          <label>&lt;p&gt;The roles adjustments are in the range from package selection up
+to disk partitioning. By choosing a CASP role, the system is
+configured accordingly to match the use case of the role. &lt;/p&gt;</label>
         </roles_help>
-        <normal_role>
+        <dashboard_role>
+          <!-- TRANSLATORS: a label for a system role -->
+          <label>Admin Dashboard</label>
```

Comment:
> Administration Dashboard

Style question: should we use "admin" or "administration"? In any case, be consistent.